### PR TITLE
fix loop mode display and docs, fix test tool

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR fixes two issues:
- loop_mode was not displayed correctly on `aea run` and wrongly defined in docs
- test case did not work for aea projects as `packages` was a requirement when it is not

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: aea/test_tools/test_cases.py [hunk 1]
```diff
@@ -734,6 +738,6 @@ def setup_class(cls):
     def teardown_class(cls):
         """Teardown the test class."""
         cls.path_to_aea = Path(".")
-        cls.packages_dir_path = Path("..", "packages")
+        # cls.packages_dir_path = Path("..", "packages")
```
Comment (solarw) [inline aea/test_tools/test_cases.py:741]: do we need a commented line here?

Comment (DavidMinarsch) [reply aea/test_tools/test_cases.py:741]: For now, because this is only a quick fix. I did not want to remove this, can add todo

File: aea/test_tools/test_cases.py [hunk 2]
```diff
@@ -712,7 +715,7 @@ def setup_class(cls):
         """Set up the test class."""
         # make paths absolute
         cls.path_to_aea = cls.path_to_aea.absolute()
-        cls.packages_dir_path = cls.packages_dir_path.absolute()
+        # cls.packages_dir_path = cls.packages_dir_path.absolute()
```
Comment (DavidMinarsch) [inline aea/test_tools/test_cases.py:718]:
```suggestion
        # TODO: decide whether to keep optionally: cls.packages_dir_path = cls.packages_dir_path.absolute()
```

File: aea/test_tools/test_cases.py [hunk 3]
```diff
@@ -723,7 +726,8 @@ def setup_class(cls):
         cls.agent_name = agent_configuration.agent_name
 
         # this will create a temporary directory and move into it
-        BaseAEATestCase.packages_dir_path = cls.packages_dir_path
+        # BaseAEATestCase.packages_dir_path = cls.packages_dir_path
```
Comment (DavidMinarsch) [inline aea/test_tools/test_cases.py:729]:
```suggestion
        # TODO: decide whether to keep optionally:  BaseAEATestCase.packages_dir_path = cls.packages_dir_path
```

File: aea/test_tools/test_cases.py (hunk 1 above)
Comment (DavidMinarsch) [inline aea/test_tools/test_cases.py:741]:
```suggestion
        # TODO: decide whether to keep optionally:  cls.packages_dir_path = Path("..", "packages")
```

File: aea/test_tools/test_cases.py [hunk 4]
```diff
@@ -712,7 +715,7 @@ def setup_class(cls):
         """Set up the test class."""
         # make paths absolute
         cls.path_to_aea = cls.path_to_aea.absolute()
-        cls.packages_dir_path = cls.packages_dir_path.absolute()
+        # TODO: decide whether to keep optionally: cls.packages_dir_path = cls.packages_dir_path.absolute()
```
Comment (marcofavorito) [inline aea/test_tools/test_cases.py:718]: why not just `None` if `use_packages_dir = False`? It's responsibility of the user to know whether he can use it or not.

Comment (DavidMinarsch) [reply aea/test_tools/test_cases.py:718]: Well there is already quite a lot for the user to know to use this test case anyway. We need to improve them, but the quick fix is as I did so not to run into any edge cases where an optional field might cause issues.
